packages/aws-cdk/bin/cdk.ts

```diff
@@ -87,7 +87,8 @@ async function parseCommandLineArguments() {
       .option('require-approval', { type: 'string', choices: [RequireApproval.Never, RequireApproval.AnyChange, RequireApproval.Broadening], desc: 'What security-sensitive changes need manual approval' })
       .option('ci', { type: 'boolean', desc: 'Force CI detection', default: process.env.CI !== undefined })
       .option('notification-arns', { type: 'array', desc: 'ARNs of SNS topics that CloudFormation will notify with stack related events', nargs: 1, requiresArg: true })
-      .option('tags', { type: 'array', alias: 't', desc: 'Tags to add to the stack (KEY=VALUE)', nargs: 1, requiresArg: true })
+      // @deprecated(v2) -- tags are part of the Cloud Assembly and tags specified here will be overwritten on the next deployment
```

Comment:
Can you add the deprecation notices in the other places that need code removal? 

As mentioned in a different PR, it would be nice see which code blocks should be removed and if needed, re-organize code so it's easy to do this.

Reply:
Maybe also add a console output warning that this will be deprecated.